src/db_api_util.ts

```diff
@@ -1031,6 +1052,67 @@ export async function getThreadNeedsAttentionFor(
   }
 }
 
+// Return a list of ThreadInfo's for all threads needing followup
+export async function getThreadsNeedingFollowUp(
+  slackUserId: string,
+  days: number
+): Promise<ThreadInfo[]> {
+  const client = await pool.connect();
+  try {
+    const result = await client.query(
+      `WITH all_claims AS (
+        SELECT
+          user_id
+          , volunteer_slack_user_id
+          , volunteer_slack_user_name
+          , row_number () OVER (PARTITION BY user_id ORDER BY created_at DESC) AS rn
+        FROM volunteer_voter_claims
+        WHERE archived != true
+      ), latest_claims AS (
+        SELECT user_id, volunteer_slack_user_id, volunteer_slack_user_name
+        FROM all_claims
+        WHERE rn = 1
+      ), all_status AS (
+        SELECT
+          user_id, voter_status
+          , row_number() OVER (PARTITION BY user_id ORDER BY created_at DESC) AS rn
+        FROM voter_status_updates
+      ), latest_status AS (
+        SELECT user_id, voter_status
+        FROM all_status
+        WHERE rn = 1
+      )
+      SELECT
+        t.slack_parent_message_ts
+        , t.slack_channel_id
+        , t.user_id
+        , EXTRACT(EPOCH FROM now() - updated_at) as last_update_age
```

Comment:
Is this the last time the voter's thread was updated (turning needs attention on or off)? Can we do the last time the helpline sent them a message instead?

Reply:
`updated_at` is only bumped when there is a new message on the thread. It's not affected by automated messages

Reply:
remind me: is updated_at updated when a volunteer messages?

Reply:
yes.  volunteer text or voter message.